lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -621,76 +808,75 @@ def _get_seq(self, feat, contig):
+            genome_obj.ont_mappings[ontology_type]['event_index'] = len(genome_obj.ontology_events)
             if ontology_type == "GO":
                 ontology_ref = "KBaseOntology/gene_ontology"
             elif ontology_type == "PO":
                 ontology_ref = "KBaseOntology/plant_ontology"
             else:
                 ontology_ref = f"KBaseOntology/{ontology_type.lower()}_ontology"
-            self.ontology_events.append({
+            genome_obj.ontology_events.append({
                 "method": "GenomeFileUtils Genbank uploader from annotations",
                 "method_version": self.version,
-                "timestamp": self.time_string,
+                "timestamp": genome_obj.time_string,
                 "id": ontology_type,
                 "ontology_ref": ontology_ref
             })
 
-        return self.ont_mappings[ontology_type]['event_index']
+        return genome_obj.ont_mappings[ontology_type]['event_index']
 
-    def _get_ontology_db_xrefs(self, feature):
+    def _get_ontology_db_xrefs(self, feature, genome_obj):
         """Splits the ontology info from the other db_xrefs"""
         ontology = defaultdict(dict)
         db_xrefs = []
         for key in ("GO_process", "GO_function", "GO_component"):
-            ontology_event_index = self._create_ontology_event("GO")
+            ontology_event_index = self._create_ontology_event("GO", genome_obj)
             for term in feature.qualifiers.get(key, []):
                 sp = term.split(" - ")
                 ontology['GO'][sp[0]] = [ontology_event_index]
-                self.ontologies_present['GO'][sp[0]] = self.ont_mappings['GO'].get(sp[0], '')
+                genome_obj.ontologies_present['GO'][sp[0]] = genome_obj.ont_mappings['GO'].get(sp[0], '')
 
         for ref in feature.qualifiers.get('db_xref', []):
             if ref.startswith('GO:'):
-                ontology['GO'][ref] = [self._create_ontology_event("GO")]
-                self.ontologies_present['GO'][ref] = self.ont_mappings['GO'].get(ref, '')
+                ontology['GO'][ref] = [self._create_ontology_event("GO", genome_obj)]
+                genome_obj.ontologies_present['GO'][ref] = genome_obj.ont_mappings['GO'].get(ref, '')
             elif ref.startswith('PO:'):
-                ontology['PO'][ref] = [self._create_ontology_event("PO")]
-                self.ontologies_present['PO'][ref] = self.ont_mappings['PO'].get(ref, '')
+                ontology['PO'][ref] = [self._create_ontology_event("PO", genome_obj)]
+                genome_obj.ontologies_present['PO'][ref] = genome_obj.ont_mappings['PO'].get(ref, '')
             elif ref.startswith('KO:'):
-                ontology['KO'][ref] = [self._create_ontology_event("KO")]
-                self.ontologies_present['KO'][ref] = self.ont_mappings['KO'].get(ref, '')
+                ontology['KO'][ref] = [self._create_ontology_event("KO", genome_obj)]
```

Comment:
@MrCreosote 
This is probably more domain knowledge related.

In order to cover the missing annotation here and below, we need to run genbank files whose ref/ontology starts with `KO`, `COG`, `PFAM`,  and `TIGRFAM`

Reply:
Oof.

Are the changes here literally just changing the code to operation on genome_obj rather than self?

Reply:
yeah.

Reply:
Ok, as much as I dislike it that's too much to expect. Can you please read through the code call tree from those points and double check everything is right?

Reply:
operation wise just changed self to genome_object and prepare for parallelization.
I have not seen anything weird jumped out.

Reply:
all covered! 👍